Mathlib/NumberTheory/ModularForms/CongruenceSubgroups.lean

```diff
@@ -88,6 +88,14 @@ theorem Gamma_zero_bot : Gamma 0 = ⊥ := by
     simp [h]
 #align Gamma_zero_bot Gamma_zero_bot
 
+lemma ModularGroup_T_pow_mem_Gamma (N M : ℤ) (hNM : N ∣ M) :
+    (ModularGroup.T ^ M) ∈ _root_.Gamma (Int.natAbs N) := by
```

Comment:
Hmm, should `Gamma` really be in the root namespace? I think it should be namespaced. (But not in this PR.)

Reply:
Ah yes I agree, I'll make a new PR after this and put it in a namespace.